cmake/FindReadline.cmake

```diff
@@ -19,56 +19,57 @@
 #  Readline_LIBRARY          The readline library.
 
 find_path(Readline_ROOT_DIR
-    NAMES include/readline/readline.h
-    PATHS /opt/local/ /usr/local/ /usr/
-    NO_DEFAULT_PATH
+  NAMES include/readline/readline.h
+  PATHS /opt/local/ /usr/local/ /usr/
+  NO_DEFAULT_PATH
 )
 
 find_path(Readline_INCLUDE_DIR
-    NAMES readline/readline.h
-    PATHS ${Readline_ROOT_DIR}/include
-    NO_DEFAULT_PATH
+  NAMES readline/readline.h
+  PATHS ${Readline_ROOT_DIR}/include
+  NO_DEFAULT_PATH
 )
```

Comment:
All these seem to be unchanged apart from whitespace. Please refrain.

Reply:
@danrmiller the build bot doesn't have libreadline. So that's expected output.

@moneromooo-monero yes I know. Apologies. I find editing cmake files a PITA and re-indenting helped me. The file is so small I hoped this wouldn't hide the actual changes in review.

Reply:
@danrmiller I stand corrected! It looks like something else. Let me try adding tinfo.